components/links/PeekabooLink.tsx

```diff
@@ -73,7 +72,10 @@ export function PeekabooLink({
       <AnimatePresence mode="wait">
         {isOpen && (
           <HoverCard.Portal forceMount>
-            <HoverCard.Content asChild>
+            <HoverCard.Content
+              asChild
+              collisionBoundary={[250]}
```

Comment:
```suggestion
              collisionPadding={250}
```
I think it should be padding instead, right?
https://www.radix-ui.com/primitives/docs/components/hover-card

Reply:
Sorry, I also realized after resolving the conversation, that `collisionPadding ` is correct. 😂